app/assets/src/components/views/SampleUploadFlow/BasespaceSampleImport.jsx

```diff
@@ -13,6 +13,7 @@ import {
 import { withAnalytics } from "~/api/analytics";
 
 import { openBasespaceOAuthPopup } from "./utils";
+import { NO_PROJECT_ERROR } from "./constants";
```

Comment:
This can also be confusing for future developers. I would do something like `NO_TARGET_PROJECT_ERROR`.